UnityAds/src/main/java/com/mopub/mobileads/UnityRouter.java

```diff
@@ -24,23 +24,38 @@
 
     private static final UnityInterstitialCallbackRouter interstitialRouter = new UnityInterstitialCallbackRouter();
 
-    static boolean initUnityAds(Map<String, String> serverExtras, Activity launcherActivity) {
-        initGdpr(launcherActivity);
+    static boolean initUnityAds(Map<String, String> serverExtras, Context context) {
+        return initUnityAds(serverExtras, context, new IUnityAdsInitializationListener() {
+            @Override
+            public void onInitializationComplete() {
+
+            }
+
+            @Override
+            public void onInitializationFailed(UnityAds.UnityAdsInitializationError unityAdsInitializationError, String s) {
+
+            }
+        });
+    }
+
+    static boolean initUnityAds(Map<String, String> serverExtras, Context context, IUnityAdsInitializationListener initializationListener) {
+        initGdpr(context);
 
         String gameId = serverExtras.get(GAME_ID_KEY);
         if (gameId == null || gameId.isEmpty()) {
             MoPubLog.log(CUSTOM, ADAPTER_NAME, "gameId is missing or entered incorrectly in the MoPub UI");
             return false;
         }
-        initMediationMetadata(launcherActivity);
-        
+        initMediationMetadata(context);
+
         boolean testMode = false;
         boolean enablePerPlacementLoad = true;
-        UnityAds.initialize(launcherActivity, gameId, interstitialRouter, testMode, enablePerPlacementLoad);
+        UnityAds.addListener(interstitialRouter);
```

Comment:
do we still need to add listener?

Reply:
@jindoujiao yes, for extended listener

Reply:
Still kept this since we are still using `interstitialRouter` to keep track the delegates